quickstarts/azure-openai-assistants/scripts/create-azure-resource.sh

```diff
@@ -0,0 +1,94 @@
+# Azure CLI: Create an OpenAI resource
```

Comment:
Did you consider writing a bicep file instead? It is much simpler and has autocompletion features.